aspnetcore/fundamentals/logging/index.md

```diff
@@ -796,15 +796,76 @@ On Linux, this provider writes logs to */var/log/message*.
 logging.AddDebug();
 ```
 
-### EventSource provider
+### Event Source provider
 
-For apps that target ASP.NET Core 1.1.0 or later, the [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing. On Windows, it uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803). The provider is cross-platform, but there are no event collection and display tools yet for Linux or macOS.
+The [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing cross-platform. On Windows, the provider uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803).
 
 ```csharp
 logging.AddEventSourceLogger();
 ```
 
-A good way to collect and view logs is to use the [PerfView utility](https://github.com/Microsoft/perfview). There are other tools for viewing ETW logs, but PerfView provides the best experience for working with the ETW events emitted by ASP.NET Core.
+The Event Source provider is added automatically when `CreateDefaultBuilder` is called to build the host.
+
+::: moniker range=">= aspnetcore-3.0"
+
+#### dotnet trace tooling
+
+The [dotnet-trace](/dotnet/core/diagnostics/dotnet-trace) tool allows you to consume Event Source provider data.
+
+General syntax:
+
+```dotnetcli
+dotnet trace collect -p {pid} 
+    --providers Microsoft-Extensions-Logging:{Keyword}:{Event Level}
+        :FilterSpecs=
+            {Logger Category 1}:{Event Level 1};
+            {Logger Category 2}:{Event Level 2};
+            ...
+            {Logger Category N}:{Event Level N}
+```
+
+| Keyword | Description |
+| :-----: | ----------- |
+| 1       | Log meta events about the `LoggingEventSource`. Doesn't log events from `ILogger`). |
+| 2       | Turns on the `Message` event when `ILogger.Log()` is called. Provides information in a programmatic (not formatted) way. |
+| 4       | Turns on the `FormatMessage` event when `ILogger.Log()` is called. Provides the formatted string version of the information. |
+| 8       | Turns on the `MessageJson` event when `ILogger.Log()` is called. Provides a JSON representation of the arguments. |
+
+| Event Level | Description     |
+| :---------: | --------------- |
+| 0           | `LogAlways`     |
+| 1           | `Critical`      |
+| 2           | `Error`         |
+| 3           | `Warning`       |
+| 4           | `Informational` |
+| 5           | `Verbose`       |
+
+`FilterSpecs` entries for `{Logger Category}` and `{Event Level}` represent additional log filtering conditions. Separate `FilterSpecs` entries with a semicolon (`;`).
+
+Example:
+
+```dotnetcli
+dotnet trace collect -p {pid} --providers Microsoft-Extensions-Logging:4:2:FilterSpecs=Microsoft.AspNetCore.Hosting*:4
+```
+
```

Comment:
We should show a screenshot of the logs in perfview (with vs coming soon)

Reply:
We try to avoid screenshots. We can't maintain them. VS and Azure UIs change regularly, so they go stale quite fast. It's best if we can point to a Perfview doc set and let them show their own UI, if possible.

Reply:
I think its useful to show how to find them because it's not obvious how to find them today.

Reply:
Do you have a screenshot of what you would like to show?

Reply:
Nevermind my last question ... I'm getting the screenshot and some nice updates to this new content. *Stand-by!*